sites/public/src/pages/account/dashboard.tsx

```diff
@@ -65,26 +55,42 @@ function Dashboard(props: DashboardProps) {
         <section className="bg-gray-300 border-t border-gray-450">
           <div className="max-w-5xl mx-auto md:py-8">
             <SiteAlert type="success" className="md:mb-8" timeout={30000} />
-
-            <div className="flex flex-wrap relative">
-              <h1 className={"sr-only"}>{t("nav.myDashboard")}</h1>
-              <DashBlocks>
-                <DashBlock
-                  href="/account/applications"
-                  title={t("account.myApplications")}
-                  subtitle={t("account.myApplicationsSubtitle")}
-                  icon={<HeaderBadge />}
-                  dataTestId={"account-dashboard-applications"}
-                ></DashBlock>
-                <DashBlock
-                  href="/account/edit"
-                  title={t("account.accountSettings")}
-                  subtitle={t("account.accountSettingsSubtitle")}
-                  icon={settingsIcon}
-                  dataTestId={"account-dashboard-settings"}
-                ></DashBlock>
-              </DashBlocks>
-            </div>
+            <h1 className={"sr-only"}>{t("nav.myDashboard")}</h1>
+            <Grid spacing="lg" className={styles["account-card-container"]}>
+              <Grid.Row columns={2}>
+                <Grid.Cell>
+                  <AccountCard
```

Comment:
I don't know if I missed it in your write up of the PR, but I noticed the two cards are not of the same height. 
![image](https://github.com/bloom-housing/bloom/assets/42942267/12b712cc-0858-4ab5-bd4f-919349e111d7)
I want to make sure that was intentional as the figma shows them the same height

Reply:
@emilyjablonski @ludtkemorgan I ended up treating the "Ready for Dev" figma page as a source of truth for this which had the cards at unique heights. I'll circle cack with Jesse though and see if any updates should be made in the next PR around lottery filtering
<img width="1164" alt="Screenshot 2024-01-29 at 9 39 50 AM" src="https://github.com/bloom-housing/bloom/assets/53269332/fc58ffc9-2d12-41a3-bf8a-2a7eacc983fe">